cumulus/parachains/runtimes/collectives/collectives-westend/Cargo.toml

```
@@ -0,0 +1,226 @@
+sp-runtime = { path = "../../../../../substrate/primitives/runtime", default-features = false}
+sp-session = { path = "../../../../../substrate/primitives/session", default-features = false}
+sp-std = { path = "../../../../../substrate/primitives/std", default-features = false}
+sp-storage = { path = "../../../../../substrate/primitives/storage", default-features = false}
+sp-transaction-pool = { path = "../../../../../substrate/primitives/transaction-pool", default-features = false}
+sp-version = { path = "../../../../../substrate/primitives/version", default-features = false}
+
+# Polkadot
+pallet-xcm = { path = "../../../../../polkadot/xcm/pallet-xcm", default-features = false}
+polkadot-core-primitives = { path = "../../../../../polkadot/core-primitives", default-features = false}
+polkadot-parachain-primitives = { path = "../../../../../polkadot/parachain", default-features = false}
+polkadot-runtime-common = { path = "../../../../../polkadot/runtime/common", default-features = false}
+xcm = { package = "staging-xcm", path = "../../../../../polkadot/xcm", default-features = false}
+xcm-builder = { package = "staging-xcm-builder", path = "../../../../../polkadot/xcm/xcm-builder", default-features = false}
+xcm-executor = { package = "staging-xcm-executor", path = "../../../../../polkadot/xcm/xcm-executor", default-features = false}
+
+# Cumulus
+cumulus-pallet-aura-ext = { path = "../../../../pallets/aura-ext", default-features = false }
+pallet-message-queue = { path = "../../../../../substrate/frame/message-queue", default-features = false }
+cumulus-pallet-dmp-queue = { path = "../../../../pallets/dmp-queue", default-features = false }
+cumulus-pallet-parachain-system = { path = "../../../../pallets/parachain-system", default-features = false, features = ["parameterized-consensus-hook",] }
+cumulus-pallet-session-benchmarking = { path = "../../../../pallets/session-benchmarking", default-features = false}
+cumulus-pallet-xcm = { path = "../../../../pallets/xcm", default-features = false }
+cumulus-pallet-xcmp-queue = { path = "../../../../pallets/xcmp-queue", default-features = false }
+cumulus-primitives-core = { path = "../../../../primitives/core", default-features = false }
+cumulus-primitives-utility = { path = "../../../../primitives/utility", default-features = false }
+pallet-collator-selection = { path = "../../../../pallets/collator-selection", default-features = false }
+pallet-collective-content = { path = "../../../pallets/collective-content", default-features = false }
+parachain-info = { package = "staging-parachain-info", path = "../../../pallets/parachain-info", default-features = false }
+parachains-common = { path = "../../../common", default-features = false }
+
+[build-dependencies]
+substrate-wasm-builder = { path = "../../../../../substrate/utils/wasm-builder", optional = true }
+
+[dev-dependencies]
+sp-io = { path = "../../../../../substrate/primitives/io", default-features = false}
```

```suggestion
sp-io = { path = "../../../../../substrate/primitives/io", features = [ "std" ]}
```
For dev dependencies we can directly enable it.